examples/aditof-demo/aditofdemoview.cpp

```diff
@@ -190,6 +196,15 @@ void AdiTofDemoView::render() {
             playbackChecked = xorValue & 1;
         }
 
+        // play button group
+        int btnGroupView = separatedViewChecked << 1 | blendedViewChecked;
```

Comment:
You could add a description on what exactly is being done here. For e.g.: 'Creating a group of buttons where only one can be active at a time.'

Reply:
solved with: e7cff9e59a801ac79ecda9ecbc7053883bd30962